### PR TITLE
Milestone 8: Mobile Menu

### PR DESCRIPTION
**Pull request by GuilhermeGiachellin**
-Created a menu with JavaScript
-Used Dom to do it

### CODE REVIEW

File: scrip.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  Event.preventDefault();
+  setTimeout(() => { Displaymenu.style = 'display: flex; animation-name: close_menu_animation; animation-duration: 0.5s;'; }, 0);
+  setTimeout(() => { Displaymenu.style = 'display: none;'; }, 500);
+  for (let i = 0; i < 5; i += 1) {
+    sec[i].style = '';
+  }
+});
+
+document.querySelectorAll('a[href^="#"]').forEach((anchor) => {
+  anchor.addEventListener('click', function (e) {
```
Comment (vichuge) [inline scrip.js]: - [x] I kindly invite you to use ES6 in every part of your script, this following the [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
- No Linters error ✔️

## Aspects to improve ♻️

- [x] It looks like the open menu doesn't cover the entire screen, I kindly invite you to fix it.
image

- [x] At desktop size the navbar appears as moved to the right side, making it impossible to click the options on the navbar.
image

- [x] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scrip.js (hunk 1 above)
Comment (GuilhermeGiachellin) [reply scrip.js]: Thank you for the review! 

About the ES6, I believe we not about ES6 so I don't know how to adapt. Everything else is corrected.

Comment (vichuge) [reply scrip.js]: I agree with you, don't worry :)

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**